qucs/AUTHORS

```diff
@@ -17,12 +17,29 @@
 -- Boston, MA 02110-1301, USA.  
 --
 
-Maintainer:
-	Michael Margraf <michael.margraf@alumni.tu-berlin.de>
+Original developers:
+	Michael Margraf
+	Stefan Jahn
 
-Authors: (see THANKS file for details)
-	Michael Margraf <michael.margraf@alumni.tu-berlin.de>
-	Stefan Jahn <stefan@lkcc.org>
-	Jens Flucke <jens.flucke@gmx.de>
-	Gopala Krishna A <krishna.ggk@gmail.com>
-	Gunther Kraut <gn.kraut@online.de>
+Current developers:
+	Guilherme Brondani Torri
+	Mike Brinson
+	Richard Crozier
+	Bastien Roucaries
+	Frans Schreuder
+	Vadim Kuznetsov
+	Claudio Girardi
+	Felix Salfelder
+	Andr�s Mart�nez Mera
```

Comment:
Why are these symbols showing up here? Can we do something about it? I mean the file encoding or something.